src/views/start.njk

```diff
@@ -1,18 +1,114 @@
 {% extends "_base.njk" %}
 
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
 {% block content %}
   <h1 class="govuk-heading-xl">
-    Fit and Competent
+     Fit and Competent Controllers Registration
   </h1>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+
+      <p class="govuk-body">
+        You must hold Fit and Competent Controller status if you wish to control deer:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          Out of season (under a specific <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=35">5(6)</a> authorisation of the Deer Act)
+        </li>
+        <li>
+          At night (under the <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=36">18(2)</a> authorisation of the Deer Act)
+        </li>
+        <li>
+          Under the <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=34">general authorisation</a> (if you don’t meet the criteria below)
+        </li>
+      </ul>
+
+      <p class="govuk-body">
+        ! You can use this service to apply, amend, renew or surrender your status
+      </p>
+
+      <form method="post">
+        {{ govukButton({
+          text: "Continue",
+          classes: "govuk-!-margin-top-2 govuk-!-margin-bottom-8 naturescot-forward-button",
+          isStartButton: true
+        }) }}
+      </form>
+
       <p class="govuk-body">
-        Use this service to apply for a Fit and Competent licence
+        ! Any of the following who wish to control deer under the <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="http://gateway.snh.gov.uk/go/?id=34">General Authorisation</a> (to prevent damage to improved agricultural land
+        or enclosed woodland) do not need to be registered:
+      </p>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          Owner or occupier
+        </li>
+        <li>
+          An employee of the owner or occupier (if authorised by them)
+        </li>
+        <li>
+          Any other resident on the land
+        </li>
+      </ul>
```

Comment:
I'm not sure if this should/can be content within the warning component, but it's directly related to it, so it might be worth a shot.

Reply:
Resolved in c452157362c04c6d2a93f2e9c4e46915d91d02eb and 04c66972eb4fc629e8d85238e7a6eb75098eda8a.